calico_versioned_docs/version-3.28/releases.json

```
@@ -1,4 +1,56 @@
 [
+  {
+    "title": "v3.28.1",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "registry": "quay.io",
+      "version": "v1.34.0"
```

```suggestion
      "version": "v1.34.2"
```

should be 1.34.2 right @danudey  ?